debezium-server-iceberg-sink/src/main/java/io/debezium/server/iceberg/IcebergConfig.java

```diff
@@ -99,4 +99,7 @@ public interface IcebergConfig {
   @WithDefault("false")
   boolean nestedAsVariant();
 
+  @WithName("debezium.sink.iceberg.partition-spec")
+  Optional<String> partitionSpec();
+
```

Comment:
could we change this to `Optional<List<String>>`  to accept list of partition pecs, multiple columns?  
example: 
`hour(timestamp),bucket(user_id\,16)`